benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/README.md

```diff
@@ -16,6 +16,22 @@ If you are using Bash and the Gnu compiler then:
 
 should build the `nemolite2d.exe` binary.
 
+Note that in some platforms, like FPGAs, using a JIT solution for the
+OpenCL kernels is not feasible. In this case the OpenCL kernel binary
+has to be compiled ahead of time with the ``device_binary`` Makefile
+target and the proper environment loaded. For example:
+
+    > source ../../../../../compiler_setup/xilinx.sh
+    > make device_binary
+
+Since this process can take a long time it is convenient to launch
+the building process detach from the terminal session with:
+
+    > make device_binary_nohup
+
+There are equivalent ``device_binary_tasks`` and ``device_binary_tasks_nohup``
+to compile a task-based version of the OpenCL kernels.
```

Comment:
There is the problem size constraint, which I added a note about now.

Also it has to be compiled with the (tasks_optimization=.true.), which I added a note in the "Running" section below.

I the future this and the no_copy_optimization could be a runtime environment variable to make it easier to experiment with, but since both still have some rough edges (work with specific problem size and number of iteration) I wanted to still keep them as an 'internal option' for now.